smartcar-auth/src/main/java/com/smartcar/sdk/SmartcarAuthRequest.java

```diff
@@ -101,8 +101,8 @@ protected ResponseType getResponseType() {
      *
      * @return The Boolean value
      */
-    protected Boolean getDevelopment() {
-        return development;
+    protected Boolean getTestMode() {
```

Comment:
this is a breaking change

Reply:
@snggeng can we leave in the `getDevelopment()` method in, but just add an [`@Deprecated`](https://docs.oracle.com/javase/1.5.0/docs/guide/javadoc/deprecation/deprecation.html) annotation (and javadoc tag)?

The implementation can just return the result of `getTestMode()`/`testMode`.

Reply:
hmm that shouldn't be the case...

Reply:
@snggeng Ah! that actually makes sense, this method is package `protected` so its not publicly accessible. So, it would just be left in for us.

Reply:
@gurpreetatwal @snggeng we should be able to just kill this method if it's just internally by us